src/modules/cloudflare/turnstile/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface ITurnstileVerificationResponse {
+  success: boolean;
+  challenge_ts?: string;
+  hostname?: string;
+  'error-codes'?: string[];
+  metadata: { interactive: boolean };
+}
```

Comment:
We didint use the prefix I on the project, please could you remove this ? -> TurnstileVerificationResponse

Reply:
prefix removed